<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-7PPO5N6" failures="0" tests="2" name="Login Test Functionlity" time="16.924" errors="0" timestamp="2025-05-05T17:12:53 IST">
  <testcase classname="tests.BaseTest" name="@AfterMethod tearDownTest" time="0.002">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)&amp;quot; because the return value of &amp;quot;utility.ExtentManager.getTest()&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because the return value of "utility.ExtentManager.getTest()" is null
at tests.BaseTest.tearDownTest(BaseTest.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod tearDownTest -->
  <testcase classname="tests.BaseTest" name="@BeforeMethod setUp" time="-1.746445373589E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase classname="tests.BaseTest" name="@AfterMethod tearDownTest" time="-1.746445373597E9">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDownTest -->
  <testcase classname="tests.LoginTest" name="loginTest" time="13.169"/>
  <testcase classname="tests.LoginTest" name="loginTest" time="0.0">
    <skipped/>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Login Test Functionlity -->
